README.md

```diff
@@ -267,8 +267,9 @@ Author Information
 
 This role was written and contributed to by the following people:
 
-[Enis Afgan](https://github.com/afgane)  
-[Dannon Baker](https://github.com/dannon)  
-[Simon Belluzzo](https://github.com/simonalpha)  
-[John Chilton](https://github.com/jmchilton)  
-[Nate Coraor](https://github.com/natefoo)  
+[Enis Afgan](https://github.com/afgane)
```

Comment:
Trivial request, sorry. Can you retain the two spaces at the end of each line? Without those spaces in markdown these collapse to a single line.

Reply:
Trivial requests are always welcome :) Would it be ok to just make it a proper markdown list?

Reply:
Alternatively these could be a list (since they are, like, a list of things).